### ISSUE TITLE
Auto-split VMD

### ISSUE THREAD

**Issue by Enieu** [ButzYung/SystemAnimatorOnline#6]
MMD can only import 20000 keyframes, automatically splitting the vmd file when it reaches 20000 keyframes would allow for longer recordings to be used.
![image](https://user-images.githubusercontent.com/121030447/235964078-bb721b75-86e2-45ef-8837-449cd701cc6d.png)

**Comment by ButzYung** [comment]
Yeah I am aware of this issue. I will consider making the auto-split feature.
